config_parser/src/lib.rs

```diff
@@ -0,0 +1,217 @@
+use proc_macro::TokenStream;
+use quote::{quote, ToTokens};
+use syn::{Data, Expr};
+
+struct Arg<'a> {
```

Comment:
Why do you have to do this, why cant you do the same as yellowstone-grpc justing using serde parse. It would be simpler to maintain rather than doing some custom stuff ?

https://github.com/rpcpool/yellowstone-grpc/blob/master/yellowstone-grpc-geyser/src/config.rs

Reply:
This does not even enable to create sections, that we had discussed about.